docs/providers/aws/guide/serverless.yml.md

```diff
@@ -287,12 +287,13 @@ functions:
               paths:
                 paramName: true # mark path parameter as required
               headers:
-                headerName: true # mark header required
+                headerName: true # mark header as required
                 custom-header: # Optional add a new header to the request
                   required: true
                   mappedValue: context.requestId # map the header to a static value or integration request variable
               querystrings:
-                paramName: true # mark query string
+                paramName: true # mark query string as required
+                anotherParamName: false # mark query string as optional
```

Comment:
Will it have any effect? (does it makes sense to mark query string params as optional?)